test/powershell/Modules/Microsoft.PowerShell.Security/ExecutionPolicy.Tests.ps1

```diff
@@ -68,7 +68,7 @@ try {
                     (Get-Help -Name Get-Disk -ErrorAction Stop).Name | Should -Be 'Get-Disk'
                 }
                 catch {
-                    $_.ToString | Should -Be null
+                    $_.ToString | Should -Be $null
```

Comment:
Can you do `Should -BeNull` here?

Reply:
The test was authored by @TravisEz13:

https://github.com/PowerShell/PowerShell/pull/1336/commits/5669e14dffe04ce8c9b1d5a7f2a35cb13f55d305#diff-ad17ca86adf9f3b3504151c96bb227c0f6c0cafaa2d98fafe0aedad38b32ee7eR30

Reply:
When the test was committed (before https://github.com/PowerShell/PowerShell/commit/e6c037c666ac35ea16d5e9920688f59c756de038), `Get-ExecutionPolicy` used to throw `PlatformNotSupportedException` on non-Windows platforms. Although in this case the test would be skipped...

Reply:
`$_.tostring` is not calling a method, it's trying to access a property named `tostring`. That's of course incorrect.
I assume the original purpose is calling `tostring()` method, and then `$_.tostring()` is guaranteed to not be $null.
I suggest we just remove this catch block.

Reply:
This test dates to [SD/725290](https://github.com/PowerShell/PowerShell/releases/tag/SD%2F725290). I wouldn't like to assume what intent @TravisEz13 had with this test.

Reply:
@TravisEz13 Can you please comment on the test in cache block here?

Reply:
This was part of a bulk port effort.  I can only guess at the purpose, but I would guess that @daxian-dbw  is correct.

Reply:
Thanks @TravisEz13. I will make the change accordingly.